spec/factories/availabilities.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :availability do
+    association :person
```

Comment:
I would prefer to actually _not_ do this. I have found that having the Factory create the associations can dramatically slow the test suite, as it's creating objects you may not need in that test. Also, it interferes with creating stubs/mocks.
Does that make sense ?
Thanks